Resources/Prototypes/_DV/CosmicCult/Objects/cosmicweapons.yml

```
@@ -207,7 +207,8 @@
     count: 1
   - type: MeleeWeapon
     resistanceBypass: true
-    wideAnimationRotation: -135
+    wideAnimationRotation: -170 # Changed to -170
+    swingLeft: true # Added this
```

No change comments needed in _DV
```suggestion
    wideAnimationRotation: -170
    swingLeft: true
```